index.html

```diff
@@ -47,7 +47,7 @@
             <div class="letter-item" data-order="7">e</div>
         </div>
  
-        <!-- Search form construction a-->
+        <!-- Search form construction a -->
```

Comment:
Why the hell are you adding space in pull request ?!

Reply:
Because I wanted to